azure-pipelines.yml

```diff
@@ -113,14 +113,14 @@ stages:
               versionSpec: '$(python.version)'
             displayName: 'Use Python $(python.version)'
 
-          - bash: python -m pip install --upgrade pip==20.2.4
+          - bash: python -m pip install --upgrade pip==21.0.1
```

Comment:
Just checking - is it ok to update pip now because we have the new constraints-dev file and that gets us around dependency issues?

Reply:
This is under test.  If does not work, will revert.